+++
draft = false
title = "python3-exceptiongroup 1.1.3-1"
version = "1.1.3-1"
date = "2023-11-03T22:40:30"
categories = ['devel-extra']
upstreamurl = "http://pypi.python.org/pypi/exceptiongroup"
arch = "x86_64"
size = "28376"
usize = "138027"
sha1sum = "27d2defb40cc48a393c28cb4f0243232d53241d4"
depends = "['python3']"
+++
Backport of PEP 654 (exception groups)